### PR TITLE
[Docs] Add packaging instructions for universal macos apps

### PR DESCRIPTION
**Pull request by harry-cpp**

### CODE REVIEW

File: Documentation/articles/packaging_games.md [hunk 1]
```diff
@@ -40,8 +43,23 @@ YourGame.app                    (this is your root folder)
         - Resources
             - Content           (this is where all your content and XNB's should go)
             - YourGame.icns     (this is your app icon, in ICNS format)
-        - MacOS                 (this is where your game belongs, except for content files)
-        - Info.plist             (the metadata of your app)
+        - MacOS
+            - amd64             (this is where your game executable for amd64 belongs, place files from the osx-x64/publish directory here)
+            - amd64             (this is where your game executable for arm64 belongs, place files from the osx-arm64/publish directory here)
```
Comment (davidhesselbomvaltech) [inline Documentation/articles/packaging_games.md:48]: Two folders named amd64?

[the remark this answers is not in the document]
Comment (harry-cpp) [reply Documentation/articles/packaging_games.md:48]: ups, yes

**Comment by rafrafek** [comment]
It is merged but I can't see it on the website. Please publish it because Stardew Valley is not available for arm64 m1 Macs and I hope this information will help the game developer to compile it for native m1 and Rosetta will not be required 🙏

image

**Comment by rafrafek** [comment]
In the realm of code, a merger's tale,
Yet on the web, it seems to set sail.
"Release it now," the call's so clear,
Stardew Valley needs M1's cheer.

For arm64 M1 Macs, we yearn,
This knowledge to the devs, we discern.
A native touch, Rosetta's weight we shed,
With hope that soon, the game's path will spread.

In unity, let's code this dream,
A place where M1's power will gleam.
So publish it, the world to see,
Stardew Valley, M1's jubilee! 🌟

**Comment by harry-cpp** [comment]
The new website is in development right now, and all the base website code and docs have been moved to: https://github.com/MonoGame/monogame.github.io

You can find the nightly docs published at: https://monogame.github.io/articles/packaging_games.html#build-and-packaging-for-macos

**Comment by rafrafek** [comment]
Thank you!

**Pull request merged**